VirtualH89/Src/NilMemory8K.h

```diff
@@ -0,0 +1,29 @@
+///
+/// \file NilMemory8K.h
+///
+/// Implementation of an 8K page of non-existent memory. Reads 0, writes ignored.
+///
+/// \date Mar 05, 2016
+/// \author Douglas Miller
+///
+#ifndef NILMEMORY8K_H_
+#define NILMEMORY8K_H_
+
+#include "Memory8K.h"
+
+class NilMemory8K: public Memory8K
```

Comment:
Since this class doesn't need 8k of buffer, a cleaner design would have a new base class that both Memory8K and this class derive from, which doesn't allocate space. The readByte would be pure virtual in the base class, and this class would just have it return 0.